src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function adder(number) {
+    if (arguments.length < 1) {
+      const result = sum;
+
+      sum = 0;
+
+      return result;
+    } else {
```

Comment:
```suggestion
    } 
```
Тут можна без else, адже if містить всередині return